packages/account/src/Sections/Security/ConnectedApps/connected-apps-info.scss

```diff
@@ -0,0 +1,6 @@
+.connected-apps-list {
+    list-style: auto;
+    font-size: 10px;
+    padding: 15px;
+    gap: 5px;
```

Comment:
you can use `rem` instead of `px`.
can font-size be passed as prop.
please refer Figme file for correct values.